Src/Extern/SUNDIALS/AMReX_NVector_MultiFab.H

```diff
@@ -0,0 +1,107 @@
+/*--------------------------------------------------------------------
+  Time Integration and Nonlinear Solvers
+  Hands-on Lessons with SUNDIALS + AMReX
+  2019 Argonne Training Program in Extreme-Scale Computing
+
+  Authors (alphabetical):
+    David Gardner (gardner48@llnl.gov)
+    John Loffeld (loffeld1@llnl.gov)
+    Daniel Reynolds (reynolds@smu.edu)
+    Donald Willcox (dewillcox@lbl.gov)
+
+  --------------------------------------------------------------------
+  Header file for N_Vector wrap of AMReX 'MultiFab' structure.
+  --------------------------------------------------------------------*/
+
+#ifndef NVECTOR_MULTIFAB_H
+#define NVECTOR_MULTIFAB_H
+
+#include <stdio.h>
+#include <sundials/sundials_nvector.h>
+
+#include <AMReX_Geometry.H>
+#include <AMReX_MultiFab.H>
+#include <AMReX_Array.H>
+
+#ifdef __cplusplus  /* wrapper to enable C++ usage */
+extern "C" {
+#endif
+
+namespace amrex {
+namespace sundials {
+
+/* -----------------------------------------------------------------
+ * Vector content structure
+ * -----------------------------------------------------------------*/
+
+struct _N_VectorContent_MultiFab {
```

Comment:
We should in principle avoid names starting with `_` followed by a capital letter.  It's reserved by the C++ standard.

Reply:
Renamed struct to avoid `_` followed by a capital letter.